sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-integration.scheduler.e2e-spec.ts

```diff
@@ -302,11 +308,111 @@ describe(describeProcessorRootTest(QueueNames.IER12Integration), () => {
     );
     // Line 1 validations.
     const firstDisbursementId = numberToText(firstDisbursement.id);
+    expect(line1.length).toBe(IER_RECORD_EXPECTED_LENGTH);
     expect(line1).toBe(
-      `${assessmentId}${firstDisbursementId}${defaultApplicationNumber}12345679    242963189Jane With Really Long Mon               19980113B   MA       Some Foreign Street Addre                         New York                     SOME POSTAL CODEProgram                  Program description                               undergraduateCertificate 0001    8   0512123401234ADR1                         6${currentOfferingId}${parentOfferingId}                              2000081620001010000033330000004444000000555500000066660019100F2000060120002001ASMT20000601000010000000006000000004800000N NNN            20000602        000966867900000000000009668679003002001003000003005NNY000000000000000000000000000000000000000000000000000000000000NY0000000000000144430000000000000000000000000000000000007777000000150056000000000000000000000000000000000000000000003000000000050000000065430000009876120005500000 0000000000000000000000000000000000000000000000000000000000000000000000ASMT20000601        Completed Pending   20000816                        CSLF0000100000BCSL0000600000CSGP0000200000CSGD0000300000CSGF0000400000CSGT0000500000BCAG0000700000SBSD0000900000BGPD0000800000    0000000000`,
+      `${assessmentId}${firstDisbursementId}${defaultApplicationNumber}12345679    242963189Jane With Really Long Mon               19980113B   MA  NONENSome Foreign Street Addre                         New York                     SOME POSTAL CODEProgram                  Program description                               undergraduateCertificate 0001    8   0512123401234ADR1                         6${currentOfferingId}${parentOfferingId}                              2000081620001010000033330000004444000000555500000066660019100F2000060120002001ASMT20000601000010000000006000000004800000NNNNN            20000602        000002850000000000000000400000006002001003000003005NNY000000000000000000000000000000000000000000000000000000000000NY0000000000000144430000000000000000021515000000000000007777000000150056000000000000000000000000000000000000000000003000000000050000000065430000001700000005500000 0000000000000000000000000000000000000000000000000000000000000000000000ASMT20000601        Completed Pending   20000816                        CSLF0000100000BCSL0000600000CSGP0000200000CSGD0000300000CSGF0000400000CSGT0000500000BCAG0000700000SBSD0000900000BGPD0000800000    0000000000`,
     );
   });
 
+  it(
```

Comment:
What is the exact criteria to break the description 😄 
Minor, but it would be good to follow the should/when pattern,

Reply:
Just noticed now that all the tests in this file were written without the "when".

Reply:
I can add when.